### DISCUSSION TITLE
nedb and minimongo

### DISCUSSION THREAD

**Discussion by harryadel** [meteor/meteor#11598] [Feature Requests]
I was thinking if there's a way to use [nedb](https://github.com/seald/nedb) within minimongo. nedb contains many features which minimongo is yet to implement like $elemMatch, indexing and many more. So, it'd be nice if we build off of their work. Has anyone attempted to do this? What do you guys think?

**Comment by StorytellerCZ** [comment]
Interesting, didn't know about that project. Definitely worth a look to at least get inspiration.

**Reply by filipenevola** [reply]
I didn't know this project as well.

We would need to test it first before thinking about adoption in Meteor core but it seems to be promising.

@harryadel have you tried it? 😄

**Reply by harryadel** [reply]
No, I was merely looking into how they had done [indexes](https://github.com/seald/nedb/blob/master/lib/indexes.js) vs [meteor](https://github.com/helfer/minimongo-index/).

**Reply by filipenevola** [reply]
It would be nice to have validations from people using Minimongo so we could compare the experiences.

**Comment by StorytellerCZ** [comment]
Saw this and wanted to add a link here to it: https://github.com/kofrasa/mingo
